thorlcr/master/thmastermain.cpp

```diff
@@ -551,8 +551,18 @@ bool ControlHandler(ahType type)
 }
 
 
+static constexpr const char * defaultJson = R"!!({
+"version": "1.0",
+"Thor": {
+    "daliServers": "dali",
```

Comment:
I'm dithering about whether defaulting the dali setting is a good idea. I ended up taking it out of Roxie (as it meant standalone roxie programs did not work)